src/C4Log.cpp

```diff
@@ -37,15 +37,28 @@ StdStrBuf sFatalError;
 bool OpenLog()
 {
 	// open
-	sLogFileName = C4CFN_Log; int iLog = 2;
+#ifndef __linux__
+	sLogFileName = C4CFN_Log;
+#else
+	sLogFileName  = Config.General.XDG_CONFIG_PATH;
+	sLogFileName.Append("/");
+	sLogFileName.Append(C4CFN_Log);
+#endif
+	int iLog = 2;
 #ifdef _WIN32
 	while (!(C4LogFile = _fsopen(sLogFileName.getData(), "wt", _SH_DENYWR)))
 #else
 	while (!(C4LogFile = fopen(sLogFileName.getData(), "wb")))
 #endif
 	{
 		// try different name
+#ifndef __linux__
 		sLogFileName.Format(C4CFN_LogEx, iLog++);
+#else
+		sLogFileName = Config.General.XDG_CONFIG_PATH;
+		sLogFileName.Append("/");
```

Comment:
Why do this twice?

Reply:
Once for Clonk.log, once for Clonk[number].log. I'm sure there's a better way to do this.

Reply:
Yes, but you can cache the result of the config path with the slash appended.